Compiling zeromq library fails

I got an error when trying to compile my app. When installing the erlzmq2 dependency, it failed.

Here is the error log:
==> Entering directory `../releases/20160603075854/broker/deps/erlzmq'
==> erlzmq (compile)
make[1]: Entering directory `../releases/20160603075854/broker/deps/erlzmq/c_src'
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
Dload  Upload   Total   Spent    Left  Speed
100   414    0   414    0     0    704      0 --:--:-- --:--:-- --:--:--   706- --:--:-- --:--:--     0

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
make[1]: *** [../deps/zeromq3] Error 2
make[1]: Leaving directory `../releases/20160603075854/broker/deps/erlzmq/c_src'
ERROR: Command [compile] failed!
make:
*** [deps] Error 1

After checking, the Makefile was fetching zeromq from http://download.zeromq.org/zeromq-$(ZEROMQ_VERSION).tar.gz, but it seems like that URL is gone.

Even on the official site, it doesn't link to that URL anymore: https://zeromq.github.io/zeromq3-x/

--- a/c_src/Makefile
+++ b/c_src/Makefile
@@ -25,7 +25,7 @@ distclean:
 
 $(DEPS)/zeromq3:
 	@mkdir -p $(DEPS)
-	@curl http://download.zeromq.org/zeromq-$(ZEROMQ_VERSION).tar.gz -o $(DEPS)/zeromq-$(ZEROMQ_VERSION).tar.gz
+	@curl -L https://archive.org/download/zeromq_$(ZEROMQ_VERSION)/zeromq-$(ZEROMQ_VERSION).tar.gz -o $(DEPS)/zeromq-$(ZEROMQ_VERSION).tar.gz
 	@cd $(DEPS) && tar xzvfp zeromq-$(ZEROMQ_VERSION).tar.gz && mv zeromq-$(ZEROMQ_VERSION) zeromq3
 
 $(DEPS)/zeromq3/src/.libs/libzmq.a: $(DEPS)/zeromq3

